lib/go-atscfg/storagedotconfig_test.go

```diff
@@ -112,7 +112,7 @@ func TestMakeStorageDotConfigNoParams(t *testing.T) {
 
 	testComment(t, txt, hdr)
 
-	if count := strings.Count(txt, "\n"); count != 2 { // comment plus a blank line
+	if count := strings.Count(txt, "\n"); count != 3 { // comment header plus its blank plus a blank line
 		t.Errorf("expected one line for comment, plus one blank line (it's important to send a blank line, to prevent many callers from returning a 404), actual: '"+txt+"' count %v", count)
```

Comment:
> ... plus one blank line <ins>and another blank line</ins> ...

Reply:
I don't think we typically require code comments to be complete sentences or proper English, outside those generating documentation like GoDoc. 

I think the sentence you're suggesting also loses data, the fact that one of the blanks is associated with the comment header, and the other is unrelated, is relevant and useful information.

We could make the last clause "plus an additional, unrelated blank line." But I feel like the verbosity would make the code less readable in this case.

Reply:
That's not referring to the comment, it's the error message. Currently it says it expects "one line for the comment and one blank line" but can then print out exactly that leaving the reader puzzled how that's an error.

Reply:
Ah, I thought you meant the inline comment. Ok, I changed the error message as well.